CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ If you are already familiar with Julia itself, this blog post by Katharine Hyatt
 
 ## Learning Julia
 
-[The learning page](https://julialang.org/learning) has a great list of resources for new and experienced users alike. [This tutorial video](https://www.youtube.com/watch?v=vWkgEddb4-A) is one recommended starting point, as is the "[Invitation to Julia](https://www.youtube.com/watch?v=gQ1y5NUD_RI)" workshop video from JuliaCon 2015  ([slide materials here](https://github.com/dpsanders/invitation_to_julia)). The [Julia documentation](https://docs.julialang.org) covers the language and core library features, and is searchable.
+[The learning page](https://julialang.org/learning) has a great list of resources for new and experienced users alike. [This video tutorial by Dr Jane Herriman](https://youtu.be/8h8rQyEpiZA) is one recommended starting point and there are others which can be found on the [Julia Language's YouTube](https://www.youtube.com/channel/UC9IuUwwE2xdjQUT_LMLONoA). The [Julia documentation](https://docs.julialang.org) covers the language itself in addition to the core library features. The documentation is also searchable for your convenience.
```

Comment:
Is this just repeating information what is already on the learning page? In that case it might be best to remove everything but the first sentence? It is easier to keep just that page up to date.

Reply:
Yeah, I agree. Will remove everything but the first line.